src/eko/io/runcards.py

```diff
@@ -5,30 +5,37 @@
 Squares are consistenly taken inside.
 
 """
-from dataclasses import dataclass
+from dataclasses import dataclass, fields
 from math import nan
-from typing import Optional, Union
+from typing import List, Optional, Union
 
 import numpy as np
 import numpy.typing as npt
 
 from .. import basis_rotation as br
-from .. import interpolation
+from .. import interpolation, msbar_masses
 from .. import version as vmod
+from ..couplings import couplings_mod_ev
+from ..quantities.heavy_quarks import (
+    FLAVORS,
```

Comment:
I think we should scope here: `from bla import FLAVORS as HFLAVORS` (or even more explicit if you wish) (like wise everywhere else)

Reply:
In this case, if you prefer, I can just `from ..quantities import heavy_quarks as hq`.

Reply:
Done in d0451659042bb7729173526573625997fcdd007b